packages/global/components/site-header/contextual.marko

```diff
@@ -7,8 +7,8 @@ $ const { config, site, req } = out.global;
 $ const blockName = input.blockName || "site-header";
 
 $ const contexts = site.getAsArray("navigation.contexts");
-$ const contextualNav = contexts.find(({ when }) => asArray(when).some(path => isActiveLink(req.path, path)));
 
+$ const contextualNav = (req.path.match(/\/[0-9]+/)) ? null : contexts.find(({ when }) => asArray(when).some(path => isActiveLink(req.path, path)))
```

Comment:
I am guessing at minimum the regex should be update to \/[0-9]{8} which should match our /{8digitContentId}

@zarathustra323 or @solocommand  are we better off passing a pageType in from the wrapping document.marko call.  Meaning that in document.marko we would do something like `<global-contextual-header page-type=get(input, 'container.page.for') />` 

Then in this file we would do something like:
```
$ const pageType = input.pageType || "website-section";
$ const contextualNav = (pageType === 'website-section') ? contexts.find(({ when }) => asArray(when).some(path => isActiveLink(req.path, path))) : null;
```

Reply:
I'm not familiar with how this component is being included, but yes using the page type would be preferable to a regex test of the URL. In both cases this would fail if someone decided to create a section like `20102020-retrospective`.

Ideally this would be placed in the content layout or template, but it looks like Bizbash is set up a bit differently. I'd look into how the nav is manipulated on [allured](https://github.com/parameter1/allured-business-media-websites/blob/master/packages/shared/components/document.marko#L86) for inspiration:

```marko
<marko-web-default-page-layout type="content">
  <@above-container context-aware-header=false >
    ...
```